lib/log_decorator.rb

```diff
@@ -0,0 +1,26 @@
+require "log_decorator/version"
+require "log_decorator/log_proxy"
+require "log_decorator/logging"
+
+module LogDecorator
+  DEFAULT_PREFIX = lambda do |klass, separator, location|
+    location = location.first if location.is_a?(Array)
+    meth = location.base_label
+    meth ? "#{klass}#{separator}#{meth}" : klass
+  end
+
+  DEFAULT_LOG_LEVELS = [:debug, :info, :warn, :error]
+
+  class << self
+    attr_writer :prefix, :log_levels
```

Comment:
I don't see any test on log_levels= and I'm curious why that would be needed at all?